.pre-commit-config.yaml

```diff
@@ -6,26 +6,27 @@ default_language_version:
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+    rev: v4.4.0
     hooks:
     - id: check-yaml
     - id: check-ast
     - id: trailing-whitespace
     - id: check-toml
   - repo: https://github.com/PyCQA/isort
-    rev: 5.13.1
+    rev: 5.12.0
```

Comment:
Why are these being pinned back?

Reply:
Presumably the repo hasn't had pre-commit autoupdate enabled and is using an older version - safe to ignore, these will get sorted as part of the repo.

Reply:
Bumped it back up for merge conflict.